src/Components/Home.js

```diff
@@ -14,21 +15,28 @@ const GET_PETS = gql`
 `
 
 const Home = () => {
+  const { loading, error, data } = useQuery(GET_PETS)
+  console.log(useQuery(GET_PETS));
 
-  const { error, loading, data } = useQuery(GET_PETS)
-  console.log('data in home', data)
+  if (loading) {
```

Comment:
We also discovered that we HAD to do something with loading (and error?) otherwise the data would not load. All of errors. As soon as we added these p tags, voila! Would love to dig into that deeper to discover why exactly that is the case.